bootstrap/kubeadm/internal/controllers/kubeadmconfig_controller.go

```diff
@@ -1012,3 +1028,17 @@ func (r *KubeadmConfigReconciler) storeBootstrapData(ctx context.Context, scope
 	conditions.MarkTrue(scope.Config, bootstrapv1.DataSecretAvailableCondition)
 	return nil
 }
+
+// addCertificateExpiryAnnotation reconciles the certificate expiry time on Kubeadmconfig.
+// Sets the certificate expiration time as on annotation on the config.
+// If the annotation is already present, there is nothing to do.
```

Comment:
```suggestion
// addCertificateExpiryAnnotation sets the certificate expiration time as an 
// annotation on KubeadmConfig, if it doesn't exist already.
```
I found reconcile a bit confusing